admin/src/js/main.js

```diff
@@ -42,7 +42,8 @@ require('./controllers/translation-languages');
 require('./controllers/upgrade');
 require('./controllers/upgrade-confirm');
 require('./controllers/users');
-require('./controllers/branding');
+require('./controllers/images-branding');
+require('./controllers/images-partners');
```

Comment:
This list is in alphabetical order so it's easy to find the files and also reduces the chances of commits conflicting because everyone is updating different parts of the file.